v2.1/admin-ui-access-and-navigate.md

```diff
@@ -29,81 +33,13 @@ The main panel displays changes for each page:
 
 Page | Main Panel Component
 -----------|------------
-Cluster Overview | <ul><li>[Cluster Overview panel](admin-ui-access-and-navigate.html#cluster-overview-panel)</li><li>[Node List](admin-ui-access-and-navigate.html#node-list). [Enterprise users](enterprise-licensing.html) can enable and switch to the [Node Map](admin-ui-access-and-navigate.html#node-map-enterprise) view. </li></ul>
-Cluster Metrics | <ul><li>[Time Series graphs](admin-ui-access-and-navigate.html#time-series-graphs)</li><li>[Summary Panel](admin-ui-access-and-navigate.html#summary-panel)</li><li>[Events List](admin-ui-access-and-navigate.html#events-panel)</li></ul>
+Cluster Overview | <ul><li>[Cluster Overview panel](admin-ui-cluster-overview-page.html)</li><li>[Node List](admin-ui-cluster-overview-page.html#node-list) </li> <li>[Enterprise users](enterprise-licensing.html) can enable and switch to the [Node Map](admin-ui-cluster-overview-page.html#node-map-enterprise) view. </li></ul>
+Cluster Metrics | <ul><li>[Time Series graphs](admin-ui-access-and-navigate.html#cluster-metrics)</li><li>[Summary Panel](admin-ui-access-and-navigate.html#summary-panel)</li><li>[Events List](admin-ui-access-and-navigate.html#events-panel)</li></ul>
 Databases | Information about the tables and grants in your [databases](admin-ui-databases-page.html).
+Statements | Information about the SQL [statements](admin-ui-statements-page.html) running in the cluster .
```

Comment:
extra space between 'cluster' and the period.

Reply:
Good catch. Fixed.